adapter/src/test/java/com/twilio/kudu/sql/PaginationIT.java

```diff
@@ -207,67 +207,6 @@ public void testQueryNoRows() throws Exception {
     }
   }
 
-  @Test
-  public void testLimit() throws Exception {
```

Comment:
Can we just update the plan instead of removing these two tests.

Reply:
I updated the plans and used `-Dcalcite.debug=true` to visually verify something important. 

One of these plans now places a `KuduSortRel(offset=[4], fetch=[20], groupBySorted=[false])`. The returned `KuduEnumerable` has `sort` property set to false:

```java
/*   1 */ public org.apache.calcite.linq4j.Enumerable bind(final org.apache.calcite.DataContext root) {
/*   2 */   final com.twilio.kudu.sql.KuduScanStats v2stashed = (com.twilio.kudu.sql.KuduScanStats) root.get("v2stashed");
/*   3 */   final java.util.List v3stashed = (java.util.List) root.get("v3stashed");
/*   4 */   final java.util.List v1stashed = (java.util.List) root.get("v1stashed");
/*   5 */   return ((com.twilio.kudu.sql.CalciteKuduTable.KuduQueryable) org.apache.calcite.schema.Schemas.queryable(root, root.getRootSchema().getSubSchema("kudu"), java.lang.Object[].class, "TABLE_ASC")).query(
v1stashed /* predicates */, v3stashed /* columns */, 
20L /* Limit */, 5L /* fetch */, false /* sorted */, false /* groupBySorted */, 
v2stashed  /* scanStats */, 
(java.util.concurrent.atomic.AtomicBoolean) root.get("cancelFlag"), new org.apache.calcite.linq4j.function.Function1(){
/*   6 */       public Object apply(final Object abstractRow) {
/*   7 */         return new Object[] {
/*   8 */ (            (org.apache.kudu.client.RowResult) abstractRow).getString(0),
/*   9 */ (            (org.apache.kudu.client.RowResult) abstractRow).getTimestamp(1).toInstant().toEpochMilli(),
/*  10 */ (            (org.apache.kudu.client.RowResult) abstractRow).getString(2),
/*  11 */ (            (org.apache.kudu.client.RowResult) abstractRow).getString(3)};
/*  12 */       }
/*  13 */ 
/*  14 */     }, new org.apache.calcite.linq4j.function.Predicate1(){
/*  15 */       public boolean apply(final Object abstractRow) {
/*  16 */         return true;
/*  17 */       }
/*  18 */ 
/*  19 */     }, false);
/*  20 */ }
/*  21 */ 
/*  22 */ 
/*  23 */ public Class getElementType() {
/*  24 */   return java.lang.Object[].class;
/*  25 */ }
/*  26 */ 
/*  27 */ 
```

Reply:
We should also print the sorted flag so that this shows up in the plan
https://github.com/twilio/calcite-kudu/blob/6bcab701454734bfdf8e708d2c7f180fa93e3d15/adapter/src/main/java/com/twilio/kudu/sql/rel/KuduSortRel.java#L62